### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by SergeyTeslov**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @Akorolev1-d

### CODE REVIEW

File: WpfApp3/WpfApp3/MainWindow.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        }
+
+        
+        private void Button_Click(object sender, RoutedEventArgs e)
+        {
+            string allText = tb_text.Text;
+
+            if (tb_text.Text.Contains("  "))
+            {
+                allText = tb_text.Text.Replace("  ", " ");
```
Comment (SergeyTeslov) [inline WpfApp3/WpfApp3/MainWindow.xaml.cs:35]: Ввел 4 пробела между "Приложение "  и " три ".
image
а потом нажимаю на рассчитать и :
image

**Comment by Akorolev1-d** [comment]
Так в задании же говорилось только про двойные пробелы.
 
>Воскресенье, 1 ноября 2020, 0:20 +03:00 от SergeyTeslov < notifications@github.com >:
> 
> 
>@SergeyTeslov commented on this pull request.
>----------------------------------------------------------------------
>In  WpfApp3/WpfApp3/MainWindow.xaml.cs :
>> +    /// </summary>
>
>+    public partial class MainWindow : Window
>
>+    {
>
>+        public MainWindow()
>
>+        {
>
>+            InitializeComponent();
>
>+        }
>
>+
>
>+        
>
>+        private void Button_Click(object sender, RoutedEventArgs e)
>
>+        {
>
>+            string allText = tb_text.Text;
>
>+
>
>+            if (tb_text.Text.Contains("  "))
>
>+            {
>
>+                allText = tb_text.Text.Replace("  ", " ");
>
>
>Ввел 4 пробела между "Приложение " и " три ".
>
>а потом нажимаю на рассчитать и :
>—
>You are receiving this because you were mentioned.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
--
Александр Королёв

**Comment by SergeyTeslov** [comment]
> Так в задании же говорилось только про двойные пробелы.  
> Воскресенье, 1 ноября 2020, 0:20 +03:00 от SergeyTeslov < ***@***.*** >:     @SergeyTeslov commented on this pull request. ---------------------------------------------------------------------- In WpfApp3/WpfApp3/MainWindow.xaml.cs : > + /// </summary> + public partial class MainWindow : Window + { + public MainWindow() + { + InitializeComponent(); + } + + + private void Button_Click(object sender, RoutedEventArgs e) + { + string allText = tb_text.Text; + + if (tb_text.Text.Contains(" ")) + { + allText = tb_text.Text.Replace(" ", " "); Ввел 4 пробела между "Приложение " и " три ". а потом нажимаю на рассчитать и : — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub , or unsubscribe .
>   -- Александр Королёв


> Так в задании же говорилось только про двойные пробелы.  
> Воскресенье, 1 ноября 2020, 0:20 +03:00 от SergeyTeslov < ***@***.*** >:     @SergeyTeslov commented on this pull request. ---------------------------------------------------------------------- In WpfApp3/WpfApp3/MainWindow.xaml.cs : > + /// </summary> + public partial class MainWindow : Window + { + public MainWindow() + { + InitializeComponent(); + } + + + private void Button_Click(object sender, RoutedEventArgs e) + { + string allText = tb_text.Text; + + if (tb_text.Text.Contains(" ")) + { + allText = tb_text.Text.Replace(" ", " "); Ввел 4 пробела между "Приложение " и " три ". а потом нажимаю на рассчитать и : — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub , or unsubscribe .
>   -- Александр Королёв



> Так в задании же говорилось только про двойные пробелы.  
> Воскресенье, 1 ноября 2020, 0:20 +03:00 от SergeyTeslov < ***@***.*** >:     @SergeyTeslov commented on this pull request. ---------------------------------------------------------------------- In WpfApp3/WpfApp3/MainWindow.xaml.cs : > + /// </summary> + public partial class MainWindow : Window + { + public MainWindow() + { + InitializeComponent(); + } + + + private void Button_Click(object sender, RoutedEventArgs e) + { + string allText = tb_text.Text; + + if (tb_text.Text.Contains(" ")) + { + allText = tb_text.Text.Replace(" ", " "); Ввел 4 пробела между "Приложение " и " три ". а потом нажимаю на рассчитать и : — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub , or unsubscribe .
>   -- Александр Королёв

Больше двух пробелов это ведь нестандартная ситуация, которую надо тоже обрабатывать.
Это надо как то обработать, потому что это не корректная работа. Формат данных какой я ввожу не задается в ограничении двух пробелов. 
image

**Comment by SergeyTeslov** [comment]
что то не так выходит с точкой 
image

**Comment by SergeyTeslov** [comment]
Оценка: 5/5

**Pull request merged**